pkg/cache/snapshot_test.go

```diff
@@ -427,23 +442,38 @@ func TestSnapshotAddRemoveWorkload(t *testing.T) {
 	workloads := []kueue.Workload{
 		*utiltesting.MakeWorkload("c1-cpu", "").
 			Request(corev1.ResourceCPU, "1").
-			Admit(utiltesting.MakeAdmission("c1").Flavor(corev1.ResourceCPU, "default").Obj()).
+			Admit(utiltesting.MakeAdmission("c1").
+				Flavor(corev1.ResourceCPU, "default").
+				Resource(corev1.ResourceCPU, "1000m").
```

Comment:
change to set both the flavor and the values in a single call

Reply:
My suggestion was not so much about the number of lines but the fact that you need to manually calculate the total resource usage to put it in the test case. And if we change a value, we need to change it in two places.

If we find a way to avoid this, we can leave it for a follow up.

Reply:
In my opinion, since the flavor reference in provided manually, the the quantities should be also. What could add some limited help could be to validate the values in the `WorkloadWrapper.Admit`.

Reply:
But then `Admit` has to know how to calculate the total, which leads to the same dependencies complications.